src/root/components/navbar.js

```diff
@@ -48,23 +48,41 @@ function Navbar(props) {
     window.clearTimeout(debounceTimeoutRef.current);
 
     if (input.length === 2) {
-      let ans = [];
-      if (input in CountryMatches) {
-        let value = CountryMatches[input][0];
-        let label = CountryMatches[input][1];
-        for (let i = 0; i < value.length; i++) {
-          ans.push({value: "/countries/" + value[i], label: "Country: " + label[i]});
-        }
-      }
-      callback(ans);
+      const exceptions = {
+        'uk': ['GBR'],
+        'us': ['USA'],
+        'co': ['CIV'],
+        'em': ['ARE'],
+      };
+      const exceptionCountries = countries.data.results.filter(country => {
+        return exceptions.hasOwnProperty(input) && exceptions[input].includes(country.iso3);
+      });
+      const answers = countries.data.results.filter(country => {
+        return country.name.toLowerCase().startsWith(input.toLowerCase())
+            && country.independent
+            && !country.is_deprecated;
+      }).concat(exceptionCountries).map(country => {
+        return {
+          value: `/country/${country.id}`,
+          label: `Country: ${country.name}`
+        };
+      });
+      //if (input in CountryMatches) {
```

Comment:
@szabozoltan69 can we clear out commented code?

Reply:
Yes, I cleared it (and used "countries" instead of "country").
https://github.com/IFRCGo/go-frontend/blob/develop/src/root/components/navbar.js#L70